### DISCUSSION TITLE
Tunable XP setting for 0.25-to-1x Option Please

### DISCUSSION THREAD

**Discussion by jw48165** [chromiecraft/chromiecraft#4010] [Suggestions]
I have four toons at 60, now leveling my 5th.  My goal this time around is to not skip quests and content due to level.  However, I'm trying to figure out how to do that easily.  It involves cycling XP on or off and turning off 2x weekend XP.  Right now, when I pop a new level and a quest in my log is below that level, I turn xp off.  It's a juggling thing that's not ideal.  I think especially at this point with the server, other people may be doing the same thing as me and would find slower leveling attractive, while not having to shut it off entirely.  

So, how about a flag?

**Comment by Amrx2** [comment]
Thanks for the suggestion, it's something we will take away and discuss internally if it will be a good match for us on Chromiecraft.

Cheers,
Construct

**Comment by hofik19** [comment]
I would personally love to set my experience rate below 1x and make my journey last longer, especially when we venture through Dark Portal

**Reply by jw48165** [reply]
Exactly!

**Comment by ghost** [comment]
I came to propose (in essence) the same thing. I have seen on another server that the experience is at the player's choice within a range and I also saw that there are "modes" of play in which you slow down and include rewards for taking that path; both concepts are interesting to analyze. I would love to go x0.25 or play in "slow mode", equally this x1 server and with the option to freeze experience is already really good in itself, cheers!

**Comment by Amrx2** [comment]
Hi all, just to add that this hasn't yet been forgotten, just our priorities are now on the TBC launch following on from the relaunch of LegacyPlayers as well as the War Effort / AQ releases. Will be something we will visit during the TBC brackets and see how best to implement such a change.

Cheers,
Construct
